tests/test_restore.py

```diff
@@ -21,15 +26,80 @@ def test_positive_restore_online_backup(ansible_module):
 
     :CaseImportance: Critical
     """
+    # preparing a target dir
+    backup_dir = 'online_backup_restore'
+    stp = ansible_module.command("rm -rf /tmp/{}".format(backup_dir))
+    print(stp.values())
```

Comment:
why not logger. I try to avoid print mostly :)

Reply:
whoops, this I didn't wan to push, will fix